druid/src/event.rs

```diff
@@ -424,8 +438,22 @@ impl Event {
 }
 
 impl LifeCycle {
-    /// Whether this event should be sent to widgets which are currently not visible
-    /// (for example the hidden tabs in a tabs widget).
+    /// Whether this event should be sent to widgets which are currently not visible and not
+    /// accessible.
+    /// Therefore the hidden tabs in a tabs widget are `hidden` whereas the hidden
+    /// widgets in a scroll are not, since you can bring them into view by scrolling.
+    ///
+    /// This distinction between scroll and tabs is due to one of the main effects of
+    /// this method: determine which widgets are allowed to receive focus. As a rule
+    /// of thumb a widget count as `hidden` if it makes no sense for it to receive focus
+    /// via pressing TAB.
+    ///
+    /// If a widget changes which children are hidden it must call [`children_changed`].
+    ///
+    /// See also [`Event::should_propagate_to_hidden`]
+    ///
+    /// [`children_changed`]: crate::EventCtx::children_changed
+    /// [`Event::should_propagate_to_hidden`]: Event::should_propagate_to_hidden
```

Comment:
Maybe it makes more sense to keep this shorter, and just link to the docs for the `Event` version? It will make it easier to ensure we keep these up to date.

```suggestion
    /// Whether this event should be sent to widgets which are currently not visible and not
    /// accessible.
    ///
    /// For example, the hidden tabs in a tabs widget are `hidden` whereas the non-visible
    /// widgets in a scroll are not, since you can bring them into view by scrolling.
    ///
    /// This distinction between scroll and tabs is due to one of the main purposes of
    /// this method: determining which widgets are allowed to receive focus. As a rule
    /// of thumb a widget count as `hidden` if it makes no sense for it to receive focus
    /// in response to the user pressing the 'tab' key.
    ///
    /// If a widget changes which children are hidden it must call [`children_changed`].
    ///
    /// See also [`Event::should_propagate_to_hidden`].
    ///
    /// [`children_changed`]: crate::EventCtx::children_changed
    /// [`Event::should_propagate_to_hidden`]: Event::should_propagate_to_hidden
```

Reply:
Ok, done :)